src/contrast/deltaPhi.js

```diff
@@ -0,0 +1,34 @@
+// Delta Phi Star perceptual lightness contrast
+// The (difference between two Lstars each raised to phi) raised to (1/phi)
+// Symmetric, does not matter which is foreground and which is background
```

Comment:
Add a link to the source of this:

// See https://github.com/Myndex/deltaphistar

Reply:
Added with commit fe321fa